src/DynamicFormReadonlyValue.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <div class="col-md-6 dynamic-form-readonly-value">{{value}}</div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {DynamicControl, Option} from "./types";
+
+    interface Props {
+        formControl: DynamicControl
+    }
+
+    interface Computed {
+        value: string;
+        flattenedOptions: Option[] | null;
+    }
+
+    export default Vue.extend<{}, {}, Computed, Props>({
+        name: "DynamicFormReadonlyValue",
+        model: {
+            prop: "formControl"
+        },
+        props: {
+            formControl: {
+                type: Object
+            }
+        },
+        computed: {
+            flattenedOptions() {
```

Comment:
This could be a simple utility function rather than a computed property, as it isn't used by the component itself. `value()` could just pass in options if there are any to flatten.

Reply:
I thought this would be better as computed, since it should mean it only gets calculated once, so long as the options don't change

Reply:
You are, as usual, one step ahead of me!